### PR TITLE
Refactor styles to match design 

### PR DESCRIPTION
**Pull request by tnlitv**
#### Please tick a box ###
- [x] **fix** _(A bug fix_)

#### If you're adding a feature, is it documented in a storybook story?
- [x] Not applicable

#### Are the components you're working on reusable between brands?
- [ ] Yes _(Using variables that are defined in the default theme)_
- [ ] No _(I hard coded some values)_

#### If you're creating markup, did you add proper semantics? 
- [x] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

### CODE REVIEW

File: src/atoms/MainRecipe/ColorTextBox.js [hunk 1]
```diff
@@ -1,27 +1,16 @@
...
-	background-color: ${props => props.theme.colors[props.bgColor]};
-
-	@media screen and (max-width: ${props => props.theme.flexboxgrid.breakpoints.md}em) {
-		padding: 4rem 6rem;
-		font-size: 2.2rem;
-		line-height: 1.4;
-		text-align: left;
- 	}
+	text-align: left;
+	color: ${props => getColor(props.textColor)};
```
Comment (misund) [inline src/atoms/MainRecipe/ColorTextBox.js]: getColor returns a functions that takes props as a parameter - so here you can delete `props =>`

File: src/atoms/MainRecipe/ColorTextBox.js [hunk 2]
```diff
@@ -1,27 +1,16 @@
...
+	background-color: ${props =>  getColor(props.bgColor)};
```
Comment (misund) [inline src/atoms/MainRecipe/ColorTextBox.js]: and here as well

File: src/molecules/NumberedList.js [hunk 3]
```diff
@@ -9,8 +9,8 @@ const Ol = styled.ol`
 	padding: 0;
 `;
 
-const NumberedList = ({ items, itemProp }) => (
-	<Ol>
+const NumberedList = ({ items, itemProp, className }) => (
+	<Ol className={className}>
```
Comment (misund) [inline src/molecules/NumberedList.js]: Not sure if anyone on your side needs this for selenium tests. Ask anyone around you in Odessa.

Comment (tnlitv) [reply src/molecules/NumberedList.js]: It's not for selenium, it's for that thing that we discussed regarding mobile version. Styles for inner elements do not apply unless we do this. 

https://www.styled-components.com/docs/advanced#styling-normal-react-components

File: src/atoms/MainRecipe/ColorTextBox.js (hunk 1 above)
Comment (misund) [reply src/atoms/MainRecipe/ColorTextBox.js]: This comment is all wrong. Nevermind :)

File: src/atoms/MainRecipe/ColorTextBox.js (hunk 2 above)
Comment (misund) [reply src/atoms/MainRecipe/ColorTextBox.js]: and this one too!

**Pull request merged**